src/test/regress/expected/multi_select_distinct.out

```diff
@@ -3,6 +3,16 @@
 --
 -- Tests select distinct, and select distinct on features.
 --
+-- This test file has an alternative output because some HashAggregate
```

Comment:
Are there any coordinator plan changes here? Can't we use coordinator_plan() trick to avoid additional test output file? We really don't care much what the worker queries generate in these tests

Reply:
Makes sense, thanks